lib/tests/streamlit/config_test.py

```diff
@@ -28,10 +28,11 @@
 from streamlit import env_util
 from streamlit.ConfigOption import ConfigOption
 
+os.environ["STREAMLIT_COOKIE_SECRET"] = "chocolatechip"
```

Comment:
If we are patching the `os.environ` dict during `ConfigTest.setUp()`, why do we need modify the real `os.environ` dict?

Reply:
Thanks for the catch. Ideally we would go with patching in `ConfigTest.setUp()` but unfortunately this conflicts when running all the tests. `config.cookieSecret` is saved on first call and other tests references this config without the patch, setting the config without using the environment variable. Modifying the `os.environ` dict here so it can set before any other test will run. Cleaning up `ConfigTest.setup()`